Source/WebCore/css/StyleProperties.cpp

```diff
@@ -339,6 +339,10 @@ String StyleProperties::getPropertyValue(CSSPropertyID propertyID) const
     case CSSPropertyPageBreakBefore:
     case CSSPropertyPageBreakInside:
         return pageBreakPropertyValue(shorthand);
+    case CSSPropertyWebkitColumnBreakAfter:
+    case CSSPropertyWebkitColumnBreakBefore:
+    case CSSPropertyWebkitColumnBreakInside:
+        return columnBreakPropertyValue(shorthand);
```

Comment:
nit: I'd name this `webkitColumnBreakValue`.

Reply:
Changed to this:
- `breakInsideShorthandValue` for shorthands of `break-inside`
- `pageBreakValue` for `page-break-before` and `page-break-after`
- `webkitColumnBreakValue` for `-webkit-column-break-before` and `-webkit-column-break-after`